.github/workflows/check.yml

```diff
@@ -1,9 +1,9 @@
 name: Check Code
 
 on:
-  pull_request_target:
-    branches: [main]
+  pull_request:
   push:
+    branches: [main]
```

Comment:
I'm happy to add the pull_request key, but I don't want to limit CI to pushes to main.

Reply:
updated :robot: